### PR TITLE
added completions event

### PR DESCRIPTION
**Pull request by yonip23**

### CODE REVIEW

File: src/selectionHandler.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+const existingDefaultImportStatement = /Add default import ([\S]*) to existing import declaration from [\S]*/;
+const importStatements = [importStatement, existingImportStatement,importDefaultStatement, existingDefaultImportStatement];
+const DELAY_FOR_CODE_ACTION_PROVIDER = 800;
+
+export const COMPLETION_IMPORTS = 'tabnine-completion-imports';
+
+export async function selectionHandler(editor: TextEditor, edit, {currentCompletion, completions, position }) {
+    try {
+        const eventData = eventDataOf(completions, currentCompletion, editor, position);
+        console.log(eventData);
```
Comment (dimacodota) [inline src/selectionHandler.ts:17]: remove the log

File: src/selectionHandler.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        };
+    });
+
+    const length = currentCompletion.length;
+    const netLength = editor.selection.anchor.character - position.character;
+    const strength = resolveDetailOf(currInCompletions);
+    const origin = currInCompletions.origin;
+    const prefixLength = editor.document.getText(new vscode.Range(new vscode.Position(position.line, 0), position)).trimLeft().length;
+    const netPrefixLength = prefixLength - (currentCompletion.length - netLength);
+    const language = editor.document.fileName.split('.').pop();
```
Comment (dimacodota) [inline src/selectionHandler.ts:59]: there is a `language` property `document.languageId` you can use it to identify the language

**Review by dimacodota — APPROVED** [review]
see my comments, other than that LGTM

File: src/selectionHandler.ts (hunk 2 above)
Comment (yonip23) [reply src/selectionHandler.ts:59]: is it the file's extension?
because if not i cant use it

**Pull request merged**